Packages/manifest.json

```diff
@@ -70,6 +72,7 @@
       "name": "OpenUPM",
       "url": "https://package.openupm.com",
       "scopes": [
+        "com.openupm",
```

Comment:
deliberate? I don't see any packages that come from here

Reply:
It's on main also. I'd probably argue for keeping it in - it's actually surprising that we don't currently have any OpenUpm packages and we're sure to have them appear and disappear quite regularly (i.e. we'll add something then decide later we need a custom fork and so on)